packages/logger/test/end-to-end/helpers/split-and-parse-json-logs.js

```diff
@@ -3,7 +3,7 @@
  *
  * @param {string} logString
  *     The log string to parse.
- * @returns {Array<Object<string, any>>}
+ * @returns {Array<{[key: string]: any}>}
```

Comment:
**suggestion (non-blocking):** we could also look at adjusting array syntax as well. It's not causing warnings but it makes sure we're always using TypeScript-notation rather than JSDoc. This is a more complex example but think this works:

```suggestion
 * @returns {{[key: string]: any}[]}
```

A simpler example would be:

```diff
- @returns {Array<string>}
+ @returns {string[]}
```

I think this should be findable by searching for all instances of `Array<`.

Reply:
Thanks for this - I've also changed the Array typedefs to be more like TypeScript :)